readthedocs/builds/tasks.py

```diff
@@ -527,6 +526,7 @@ def send_email(self, email):
         """Send email notifications for build failures."""
         # We send only what we need from the Django model objects here to avoid
         # serialization problems in the ``readthedocs.core.tasks.send_email_task``
+        protocol = 'http' if settings.DEBUG else 'https'
```

Comment:
```suggestion
        protocol = 'https' if settings.PUBLIC_DOMAIN_USES_HTTPS else 'http'
```

Reply:
`PUBLIC_DOMAIN` is different from `PRODUCTION_DOMAIN`, in other parts of the code we check for debug to decide the protocol of the prod domain.

Reply:
Ah, you are right!